nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/src/main/resources/OSGI-INF/notification-operations-contrib.xml

```diff
@@ -1,9 +1,5 @@
 <?xml version="1.0"?>
 <component name="org.nuxeo.ecm.platform.ec.notification.operations" version="1.0">
   <extension target="org.nuxeo.ecm.core.operation.OperationServiceComponent" point="operations">
-
-    <operation class="org.nuxeo.ecm.platform.ec.notification.automation.SubscribeOperation" />
-    <operation class="org.nuxeo.ecm.platform.ec.notification.automation.UnsubscribeOperation" />
-
```

Comment:
You might want to put a comment saying that this contribution is kept for backward compatibility but the actual contributions have been moved to nuxeo/nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/main/resources/OSGI-INF/operations-contrib.xml (or maybe simply the new component name is enough: `org.nuxeo.ecm.core.automation.features.operations`).

Reply:
You don't need the `<extension>` part either.